app/src/main/res/drawable/bg_menu_create_conversation.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.bg_menu_create_conversation` appears to be unused
 Unused resources make applications larger and slow down builds.